submissions/kasionio/html-css-popup/index.css

```diff
@@ -0,0 +1,173 @@
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+	font-family: Arial, Helvetica, sans-serif;
+	font-size: 12px;
+	text-decoration: none;
+	list-style-type: none;
+	color: #000;
+}
+
+.nav-bar {
+	width: 250px;
+	display: inline-flex;
+	justify-content: space-around;
+	align-items: center;
+	float: right;
+	padding: 15px;
+}
+
+#nav-link__images {
```

Comment:
Good Work!
Avoid styling by #id or HTML tag. [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, separate responsibilities:

- HTML tag for markup;
- CSS class for styling;
- Id for JS logic

Reply:
Thanks a lot for the review! Changes are made